app/views/tutors/find_students.html.erb

```diff
@@ -81,7 +81,7 @@
                                                         <p> Course: <b> <%= course_obj.name %> </b>
                                                         <p> Student: <b> <%= tutee.first_name + ' ' + tutee.last_name%> </b></p>
                                                         <p> Subject: <b> <%= req.subject%> </b></p>
-                                                        <p class="last"> <button class="btn btn-primary" onclick="showEmailForm(['<%=Tutee.find(req.tutee_id).email%>', '<%=req.tutee_id%>', '<%=req.id %>' ])">Select Student</button></p>         
+                                                        <p class="last"> <button class="btn btn-primary" onclick="showEmailForm(['<%=Tutee.find(req.tutee_id).email%>', '<%=req.tutee_id%>', '<%=req.id %>', '<%=req.meeting_length %>' ])">Select Student</button></p>
```

Comment:
You should refactor the logic to the model. Set an instance variable in the controller that calls that logic and reference that instance variable in the view.